src/components/data_submission/dsStudyInformation.js

```diff
@@ -0,0 +1,145 @@
+import { useState, useEffect } from 'react';
+import { h, h2, div } from 'react-hyperscript-helpers';
+import { Notifications, isEmailAddress } from '../../libs/utils';
+import { User } from '../../libs/ajax';
+import { FormFieldTypes, FormField, FormTable } from './ds_common';
+
+const updateUserAndFields = async ({ setUser, onChange }) => {
+  const me = await User.getMe();
+  setUser(me);
+  onChange({key: 'dataSubmitterId', value: me.dacUserId});
```

Comment:
Should I be using `dacUserId` or `userId`?

Reply:
`userId` ... dac user id is deprecated in all of the APIs.